assets/dockerfiles/fx/d/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM metrue/fx-d-base
+
```

Comment:
seems like you're missing adding your source files here.
```
ADD . .
```
should fix.

Reply:
I added the COPY . . command as all other dockerfiles also using this command